lib/matplotlib/backends/backend_qt5.py

```diff
@@ -486,7 +486,7 @@ def drawRectangle(self, rect):
         # Draw the zoom rectangle to the QPainter.  _draw_rect_callback needs
         # to be called at the end of paintEvent.
         if rect is not None:
-            x0, y0, w, h = [pt / self._dpi_ratio for pt in rect]
+            x0, y0, w, h = [int(pt / self._dpi_ratio) for pt in rect]
```

Comment:
Or round? c.f. https://github.com/matplotlib/matplotlib/pull/15656#discussion_r344543106

I'm not sure myself. I think I tested 15656 with round but it caused some test failures. Did not have time to investigate.

Maybe `int()` is good enough to fix this if we don't have the capacity to work out `round` vs. `int`.

Reply:
We have looked at moving more things to `round` in other places (https://github.com/matplotlib/matplotlib/pull/8265) and it also caused failures.  I am inclined to stay with int here as I suspect that is what is used to be doing under the hood (it is just complaining about in now).

On the other hand, this is in drawing the zoom box so it probably doesn't really matter?